tests/Support/DataSet/SimpleDataSet.php

```diff
@@ -2,13 +2,13 @@
 
 declare(strict_types=1);
 
-namespace Yiisoft\Validator\Tests\Stub;
+namespace Yiisoft\Validator\Tests\Support\DataSet;
 
 use Yiisoft\Validator\DataSetInterface;
 
 use function array_key_exists;
 
-final class DataSet implements DataSetInterface
+final class SimpleDataSet implements DataSetInterface
```

Comment:
Isn't it completely the same as `ArrayDataSet`? Can use it instead then.

Reply:
done